srcpkgs/qt6-webengine/files/resolv_compat.h

```diff
@@ -0,0 +1,29 @@
+#if !defined(__GLIBC__)
+/***************************************************************************
+ * resolv_compat.h
+ *
+ * Mimick GLIBC's res_ninit() and res_nclose() for musl libc
+ * Note: res_init() is actually deprecated according to
+ * http://docs.oracle.com/cd/E36784_01/html/E36875/res-nclose-3resolv.html
+ **************************************************************************/
+#include <string.h>
+
+static inline int res_ninit(res_state statp)
+{
+	int rc = res_init();
+	if (statp != &_res) {
+		memcpy(statp, &_res, sizeof(*statp));
+	}
+	return rc;
+}
+
+static inline int res_nclose(res_state statp)
+{
+	if (!statp)
+		return -1;
+	if (statp != &_res) {
+		memset(statp, 0, sizeof(*statp));
+	}
+	return 0;
+}
+#endif
```

Comment:
Unless they actually touch `_res` internals (which would be kinda broken in most cases anyway), you can just do `#define res_ninit(r) res_init`; musl's resolver is completely stateless (maybe you can even do it as a CFLAGS define? nto sure). Should save you some trouble ;)

Reply:
This file is there since 6 years in qt5-webengine and never changed in that time. So I would call changing it more trouble :p.
But I could probably get the qtwebengine musl patchset to be more in line with the chromium one.